lib/common/scores.c

```diff
@@ -108,67 +108,34 @@ pcmk_readable_score(int score)
  *
  * \param[in] score1  First score to add
  * \param[in] score2  Second score to add
+ *
+ * \note This function does not have context about what the scores mean, so it
+ *       does not log any messages.
```

Comment:
Thank you, this is really helpful. These messages have always given me a headache... usually figurative but not always.

Reply:
Yeah I finally got annoyed enough to do something about it :)